src/RpcServer/RpcServer.cs

```diff
@@ -105,6 +107,9 @@ public void StartRpcServer()
         {
             host = new WebHostBuilder().UseKestrel(options => options.Listen(settings.BindAddress, settings.Port, listenOptions =>
             {
+                // Default value is 5Mb
+                options.Limits.MaxRequestBodySize = settings.MaxRequestBodySize;
```

Comment:
You should remove this.

Reply:
This is the same as default as it is, right?

Reply:
The default it 28,6mb, @cschuchardt88.
Maybe this should be a Math.Min as well, no?

Reply:
The default max looks like to be `30 x 1024 x 1024`

Reply:
He already has that under the line i commented on. I was just saying he doesn need the extra line
```
options.Limits.MaxRequestLineSize = Math.Min(settings.MaxRequestBodySize, options.Limits.MaxRequestLineSize);
```

Reply:
But this are different properties.

Defaults of this is  8,192 bytes (8 KB)

Reply:
oh ur right, looked at it at quick glance

Reply:
> You should remove this.

Why?

Reply:
One is the request line (GET) and the other is the body, I set both just in case if the configuration is less than 8k

Reply:
> > You should remove this.
> 
> Why?

looks good, but can you do `Math.Min` function